src/main/java/br/com/sbs/cubatech/category/Category.java

```diff
@@ -1,23 +1,33 @@
 package br.com.sbs.cubatech.category;
 
+import javax.persistence.*;
 import java.util.ArrayList;
 import java.util.List;
 import java.util.stream.Collectors;
 
 import static br.com.sbs.cubatech.validation.Validator.*;
 
+@Entity
+@Table(name = "category")
 public class Category {
 
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
     private String name;
     private String urlCode;
+
+    @Column(columnDefinition = "LONGTEXT")
     private String description;
```

Comment:
Creio que não é mais `LONGTEXT`.

```suggestion
    private String description;
```